sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -338,10 +340,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 
 <!-- LINKS -->
 [azure_subscription]: https://azure.microsoft.com/free
+[api_reference_doc]: https://aka.ms/azsdk-python-textanalytics-ref-docs
```

Comment:
This is a reference to the python docs

Reply:
oops

Reply:
The link should be more specific?
https://azure.github.io/azure-sdk-for-java/cognitiveservices.html

Reply:
https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cognitiveservices-textanalytics/1.1.0-preview.1/index.html

Reply:
Why is it still called `preview`? Should it be changed to `beta` now?

Reply:
@srnagar do you mean the link with 1.1.0-preview.1? If yes, it could be the reason that it is a track 1 library